manual/manual/Makefile

```diff
@@ -76,7 +76,7 @@ html: files
 	         -I ../tutorials -I ../../styles -I ../texstuff manual.hva \
 	         -e macros.tex ../manual.tex ; \
 	${HACHA} -tocter manual.html ; \
-
+	cat ../manual.css >> manual.css
```

Comment:
Except that I was mistaken, and the problem stemed from forgetting to escape `#`.